### ISSUE TITLE
key has a defined format for the 'valueName' placeholder that will be ignored.

### ISSUE THREAD

**Issue by Korpyc** [localizely/intl_utils#115]
Hey, I have next string:
`"page_subtitle": "{rangeTitle} • {somethingCount, plural, one{1 Apple} other{{somethingCount} Apples}}",
    "@page_subtitle": {
        "placeholders": {
            "rangeTitle": {
                "type": "String",
            },
            "somethingCount": {
                "type": "int",
                "format": "decimalPattern"
            }
        }
    },`
    
so, as a result, if somethingCount >1000, I expect to see result like this:
`rangeTitle • 12,123 Apples`

but, generator said: 
key has a defined format for the 'somethingCount' placeholder that will be ignored.
Consider using an additional placeholder for formatting purposes.

Can we fix it without adding one more, same int value, just for formatting purposes?

**Comment by lzoran** [comment]
You're right. Currently, an additional placeholder is required for the formatting purpose in ICU plural messages.
